src/client.toit

```diff
@@ -10,133 +10,244 @@ import .tcp
 import .transport
```

Comment:
It might be easier to skip reviewing this file and do a fresh review in https://github.com/toitlang/pkg-modbus/pull/8